app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/MongoSendTask.java

```diff
@@ -94,10 +101,22 @@ public Void doInBackground(String... urls) {
                                         } else {
                                             Log.wtf(TAG, "Bloodtest with ID: " + up.reference_id + " appears to have been deleted");
                                         }
+                                    } else if (type.equals(Treatments.class.getSimpleName())) {
```

Comment:
Oh, I was wondering why there was a separate method postTreatments in NS.  I haven't changed the non Wear Queues, so NS is still using the postTreatments() method.